src/tests/ftest/container/multiple_container_delete.py

```diff
@@ -91,12 +91,10 @@ def test_multiple_container_delete(self):
         #self.assertTrue(final_scm_fs == initial_scm_fs)
 
     def get_pool_space(self):
-        """Get pool free space for a given storage index
-        Args:
-            storage_index (int): pool property name. Defaults to
-                None.
+        """Get sscm and ssd pool free space
```

Comment:
```suggestion
        """Get scm and ssd pool free space
```
Typo?

Reply:
Yes, will fix it in another ticket of mine. Don't want to run the whole build for a minor change.